APCAppCore/APCAppCore/DataSubstrate/Model/APCUser+Bridge.m

```diff
@@ -310,6 +309,18 @@ - (void)signInOnCompletion:(void (^)(NSError *))completionBlock
                             self.sharingScope = APCUserConsentSharingScopeNone;
                         }
                     }
+                    
+                    NSArray *dataGroups = responseDictionary[@"dataGroups"];
+                    self.dataGroups = dataGroups;
+                    
+                    // TODO: Handle multiple consent groups with separate sub populations
+                    NSDictionary *consentStatuses = responseDictionary[@"consentStatuses"];
+                    for (id key in consentStatuses) {
+                        NSDictionary *status = [consentStatuses objectForKey:key];
```

Comment:
each subpopulation might have just one 'required' consent, but what about multiple subpopulations? If at most one 'required' consent, does that mean there will always be a required consent? I'm not sure I know enough about the responses/data model to build this correctly for more complicated multiple data groups situation so went simple for now.